refinery/core/models.py

```diff
@@ -1875,7 +1875,7 @@ def get_managed_group(self):
     def save(self, *args, **kwargs):
```

Comment:
It looks like the preferred way to validate model fields is via full_clean() and clean() model methods: https://docs.djangoproject.com/en/1.7/ref/models/instances/#django.db.models.Model.clean